packages/ember-simple-auth/node_modules/.bin/_mocha

```diff
@@ -0,0 +1 @@
+../mocha/bin/_mocha
```

Comment:
Are these checked in intentionally?

Reply:
Ugh yep, added those accidentally – removed now